### PR TITLE
brighten up selected text in navigation

### LINKED ISSUE #373: selected text visibility on label selection menu
### Unstyled Content

- [x] No other existing issue and/or pull request.
- [ ] URL of unstyled content.
- [ ] If unable to provide a URL, please report the class name of unstyled content.
- [x] Provide steps to reproduce (opening a dialog, etc).

### Steps to Duplicate

assign or unassign labels to an issue/pr when hovering over the already set labels you see selected text is too dark.

demo-a

I think class is `.label-select-menu .select-menu-item.selected` https://github.com/StylishThemes/GitHub-Dark/blob/master/github-dark.css#L1681-L1681

### PR DESCRIPTION
**Pull request by un1versal**
closes #373

** so this is my proposal** 

I think this is half decent compromise also brightened the section brightness slightly, could be better I suppose.

### CODE REVIEW

File: github-dark.css [hunk 1]
```diff
@@ -1529,8 +1529,9 @@
   a.filter-item.selected, .date-selector td.selected,
   .date-selector td.selectable-day:hover, .subnav-item:hover, .subnav-item:focus,
   .owner-select-target:hover, .owner-select-target:focus,
-  .suggester li.navigation-focus, .suggester li.navigation-focus small {
-    color: #efefef !important;
+  .suggester li.navigation-focus, .suggester li.navigation-focus small,
+  .label-select-menu .select-menu-item.selected, octicon-x {
```
Comment (silverwind) [inline github-dark.css]: Is octicon-x really the right selector? How does Github select/style it?

Comment (un1versal) [inline github-dark.css]: It appears to be (heres a demo with a purple color for added effect)

it also works with ```.octicon .octicon-x``` which is what the actual class said

if its right or wrong... you tell me ;)

demo-a

File: github-dark.css [hunk 2]
```diff
@@ -1529,8 +1529,9 @@
   a.filter-item.selected, .date-selector td.selected,
   .date-selector td.selectable-day:hover, .subnav-item:hover, .subnav-item:focus,
   .owner-select-target:hover, .owner-select-target:focus,
-  .suggester li.navigation-focus, .suggester li.navigation-focus small {
-    color: #efefef !important;
+  .suggester li.navigation-focus, .suggester li.navigation-focus small,
+  .label-select-menu .select-menu-item.selected, .octicon .octicon-x {
```
Comment (silverwind) [inline github-dark.css]: I'd suggest changing it to

```css
.select-menu-item.selected
```

And then add
```css
.select-menu-item .octicon
```
to the group
```css
fill: currentColor;
```

Also, not sure about this color change below. May I suggest just adding it to `#fff` instead?

Comment (silverwind) [inline github-dark.css]: The second part might not even be necessary. If `.octicon` proves to have enough specificity, color will just cascade down to the SVG.

Comment (un1versal) [inline github-dark.css]: > Also, not sure about this color change below. May I suggest just adding it to #fff instead?

yea definitely.

Im not sure what you mean about fill: currentColor; group, I cant see any anywhere.

Comment (silverwind) [inline github-dark.css]: All octicons (which are SVG) on the site have `fill: currentColor`, so you just need to set `color` on any parent element to color them too. If there's no wrongly colored parent in the way, that icon should get it's color inherited from `.select-menu-item.selected` ideally.

**Comment by Mottie** [comment]
Ummm, weird... is anyone else seeing this behavior in Chrome? I disabled all extensions, and it's still happening. It's not happening at all in Firefox.

scroll

**Comment by silverwind** [comment]
Side effect of a Chrome experiment maybe?

**Pull request closed without merging**